rocketpool-cli/node/status.go

```diff
@@ -312,6 +312,22 @@ func getStatus(c *cli.Context) error {
 		fmt.Println("The node is not registered with Rocket Pool.")
 	}
 
+	// Alerts
+	if cfg.EnableMetrics.Value == true && len(status.Alerts) > 0 {
+		// only print alerts if enabled; to avoid misleading the user to thinking everything is fine (since we really don't know).
+		fmt.Printf("\n%s=== Alerts ===%s\n", colorGreen, colorReset)
+		const maxItems = 3
+		for i, alert := range status.Alerts {
+			fmt.Println(alert.ColorString())
+			if i == maxItems-1 {
+				break
+			}
+		}
```

Comment:
```suggestion
		const maxItems = 3
		for i, alert := range status.Alerts {
			if i == maxItems {
				break
			}
			fmt.Println(alert.ColorString())
		}
```

upside- no need to reason about off-by-one errors
downside- one extra iteration if the compiler doesn't catch and optimize it.

this is the softest suggestion i think i've ever given though, so feel free to ignore